apps/docs/src/guide/types/bytes.md

```diff
@@ -2,14 +2,12 @@
 
 A dynamic array of byte values can be represented using the `Bytes` type, which represents raw bytes.
 
-<<< @/../../docs-snippets/src/guide/types/bytes.test.ts#bytes-1{ts:line-numbers}
-
```

Comment:
Should this have been removed?

(Please resolve as confirmation)

Reply:
@petertonysmith94 I've removed duplicate snippets from the same doc page. The first instance was only used for a complementary introduction, and I believe removing it makes the content cleaner.

### Before:
<img width="715" alt="Screenshot 2024-10-21 at 09 57 35" src="https://github.com/user-attachments/assets/5b6bde3b-5a3c-415f-937b-c64b8e991d69">


### After:
<img width="768" alt="Screenshot 2024-10-21 at 10 00 17" src="https://github.com/user-attachments/assets/92fe1c0c-ed97-41e8-904f-16a53b562ed2">